astro_2.0/features/ViewProposal/components/ProposalComments/hooks.ts

```diff
@@ -0,0 +1,141 @@
+import uniqBy from 'lodash/uniqBy';
```

Comment:
I consider we should name this file useProposalComments.ts and place it into hooks folder if needed.

Reply:
This hook is not for usage across the application like the hooks we have in hooks folder. It is only relevant for ProposalComments component and placed here just to separate logic an presentation in ProposalComments